docs/api/expansion/api-reference.md

```diff
@@ -0,0 +1,51 @@
+# Expansion service API reference
```

Comment:
we should slap a big fat "BETA" on this and note that its subject to change. the reason i say this is because the first iteration is reasonably different than this new iteration (at least the API is breaking). so i think until we feel comfortable we might mention that its beta and that the API could break until we remove beta from the label. what do you think @nilsnolde ?

Reply:
yep agreed!